compiler/src/main/proto/spine/protodata/source.proto

```diff
@@ -49,4 +49,8 @@ message ProtobufSourceFile {
     File file = 2;
 
     map<string, MessageType> type = 3;
```

Comment:
While we are here, please document these `map`s. It may be non-obvious what the keys are.